rebar.config

```diff
@@ -23,10 +23,7 @@
          {git, "git://github.com/opscode/mini_s3.git",
           {branch, "master"}}},
         {eper, ".*",
-         {git, "git://github.com/massemanet/eper.git", {branch, "master"}}},
-        {rebar_lock_deps_plugin, ".*",
-         {git, "git://github.com/seth/rebar_lock_deps_plugin.git", "master"}}
```

Comment:
actually, it's concrete.mk that takes care of that here. it's in the rebar.config.script if you look, but because it's there, having it here too confuses it.